client/src/components/StudentCard/index.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable jsx-a11y/no-static-element-interactions */
 /* eslint-disable react/prop-types */
```

Comment:
Great stuff Ola as always. One silly comment here; could it be possible for you delete eslint stuff before any PR.